.gitignore

```diff
@@ -12,3 +12,4 @@ Thumbs.db
 /.project
 /build/
 /out/
+/.idea/
```

Comment:
I use IntelliJ and it seems to think that putting more files in the source tree is a good idea...